tools/checked-c-convert/ProgramInfo.h

```diff
@@ -150,14 +143,20 @@ class PointerVariableConstraint : public ConstraintVariable {
   // If for all U in arrSizes, any U -> (a,b) where a = O_SizedArray or 
   // O_UnSizedArray, arrPresent is true.
   bool arrPresent;
+  // Is there an itype associated with this constraint? If there is, how was it
```

Comment:
There should be comments on the public accessors too.

Reply:
I agree, I added it.